### DISCUSSION TITLE
Delete user in admin page problem

### DISCUSSION THREAD

**Discussion by HowardLu132** [dani-garcia/vaultwarden#4605] [Q&A] [answered]
![屏幕截图 2024-06-01 225649](https://github.com/dani-garcia/vaultwarden/assets/68549891/e2cb2a27-1a35-426b-872a-f557ca376b50)
Hi, I want to delete user in admin page, but when I click Delete User button, there is no response of anything.

**Comment by BlackDex** [comment]
Are there any errors in the developer console?

**Reply by HowardLu132** [reply]
Thank you for your reply.
I probably located the error in log but I cannot make a proper troubleshooting.
![IMG_0800](https://github.com/dani-garcia/vaultwarden/assets/68549891/ae8a3e5b-f75a-4426-9c67-7a4d028a8849)
There seems have lost a .slim.js file or something.

**Comment by BlackDex** [comment, marked as answer]
Check your reverse proxy if it caches files, and see of a restart helps. Also, try to clear the browser cache. Or check in a private/incognito browser.

You also might have adjusted some templates, since our interface is English only. That might indicate you are overriding some of the admin files which now point to the wrong files, as those are updated.

**Reply by HowardLu132** [reply]
Thank you
I probably found the problem source:I used the templates to change the admin page according to https://github.com/zituoguan/vaultwarden-lang-zh_CN which seems did not update so it goes wrong.
Appreciate for your help!
